### ISSUE TITLE
Not work on my macOS Catalina 10.15.3

### ISSUE THREAD

**Issue by kamusis** [LeSchlongLong/alfred-ipabuse#1]
Thanks for the workflow, but something weird here.

When I input "ip" and a space, the IP checker workflows is the only one option, looks good, but after I input any number or paste a whole ipv4 address, the IP checker is not there anyone, the only 3 option change to:
Search Google for ...
Search Amazon for ...
Search Wikipedia for ...

OS version: macOS Catalina 10.15.3
Alfred version: Alfred 4.0.8 with Powerpack

**Comment by LeSchlongLong** [comment]
@kamusis Did you enter an API Key?

**Comment by kamusis** [comment]
yep, of course I have added, I've register a free AbuseIPDB account, copy the API key from the account page,  paste it to your workflow variables screen in alfred, name is "API", value is the key.

**Comment by LeSchlongLong** [comment]
Ok! Usually there is no output when either the IP Address is invalid or it is not possible to reach abuseIPDB. I am currently on 10.15.2 and will be updating tonight to check if there is some software issue. But I doubt it. Do you have any Firewall like Little Snitch that may block access to abuseIPDBs servers? Note: It does actually take a sec until output appears **after** you entered a valid IP Address.

**Comment by kamusis** [comment]
No... I don't have any firewall installed, I connect to the internet with wifi through the router directly. I'm wonder if you can give my your personal email address, then maybe I can send you a screenshot or short video shot.

**Comment by LeSchlongLong** [comment]
Sure contact me at: leschlonglong@protonmail.com
